tox.ini

```
@@ -0,0 +1,18 @@
+[tox]
+env_list = py{39,310,311}, worker
+
+# [testenv]
+# skip_install = true
+
+[testenv:worker]
+changedir = worker
+setenv =
+    PDM_IGNORE_SAVED_PYTHON="1"
+deps = pdm
+allowlist_externals=
+    sh
+commands_pre =
+    pdm sync --dev
```

```suggestion
    pdm sync -G test
```
If the other suggestion on `dev` to `test` rename is committed

---

https://github.com/Zipstack/unstract/pull/402#discussion_r1636249776